src/views/layouts/main.handlebars

```diff
@@ -16,12 +16,13 @@
 
 	<body>
 		<!--Import jQuery before bootstrap.js-->
-		<script src="https://code.jquery.com/jquery-3.3.1.min.js"  integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="  crossorigin="anonymous"></script>
+		<script src="https://code.jquery.com/jquery-2.2.4.min.js" crossorigin="anonymous"></script>
+		<!--<script src="https://code.jquery.com/jquery-3.3.1.min.js"  integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="  crossorigin="anonymous"></script>-->
```

Comment:
we should use integrity checks, and we shouldn't leave dead code in place

Reply:
Updated as requested.

`curl https://code.jquery.com/jquery-2.2.4.min.js |sha256sum | cut -f1 -d' ' |xxd -r -p|base64`